fea-rs/src/token_tree/typed.rs

```diff
@@ -89,7 +89,7 @@ macro_rules! ast_node {
         #[derive(Clone, Debug)]
         #[allow(missing_docs)]
         pub struct $typ {
-            inner: Node,
+            pub(crate) inner: Node,
```

Comment:
we generate a `node()` method you can call, so this shouldn't be necessary.

Reply:
This was to enable construction of a test value, not reading, but perhaps that will be unnecessary once revised